system_tests/system_tests_sync/test_external_accounts.py

```diff
@@ -303,3 +305,42 @@ def test_aws_based_external_account(
                 },
             },
         )
+
+def test_pluggable_external_account(
```

Comment:
please add comments what is happening for readability, like for other tests i.e. "test_before_request_impersonation"

Reply:
Done